packages/fetchai/skills/erc1155_deploy/tests/test_handlers.py

```diff
@@ -53,7 +53,7 @@ class TestFipaHandler(ERC1155DeployTestCase):
 
     def test_setup(self):
         """Test the setup method of the fipa handler."""
-        assert self.fipa_handler.setup_class() is None
+        assert self.fipa_handler.setup() is None
```

Comment:
Without looking at code more deeply, why do we need to change the method call on the behaviour when you're just changing the test class?

Reply:
because pycharm find and replace messed this up